apis/k8ssandra/v1alpha1/k8ssandracluster_types.go

```diff
@@ -416,3 +416,14 @@ func (s *K8ssandraClusterStatus) SetCondition(condition K8ssandraClusterConditio
 func init() {
 	SchemeBuilder.Register(&K8ssandraCluster{}, &K8ssandraClusterList{})
 }
+
+type Distribution string
```

Comment:
Curious on this one, why not Distribution as a structure with name, version, etc.?

Reply:
Wouldn't it mean that `Cassandra 4.0` and `Cassandra 4.1` are considered as two different distributions?
I'm just trying to improve the typing of our `ServerType` in order to avoid repeating `cassandra` and `dse` throughout the code here, not do a refactor on how `ServerType`, `ServerVersion` and `ServerImage` are related to each other.